requests_ntlm/requests_ntlm.py

```diff
@@ -13,25 +13,34 @@ class HttpNtlmAuth(AuthBase):
     Supports pass-the-hash.
     """
 
-    def __init__(self, username, password, session=None):
+    def __init__(self, username, password, session=None, domain=None):
         """Create an authentication handler for NTLM over HTTP.
 
-        :param str username: Username in 'domain\\username' format
+        :param str username: Username. If in either 'domain\\\\username'
```

Comment:
Currently we don't render HTML docs, so it might be better to just leave it in the double form.